constants/Enchants.json

```diff
@@ -237,13 +231,13 @@
     "hardened mana": {
       "loreName": "Hardened Mana",
       "nbtName": "hardened_mana",
-      "goodLevel": 7,
+      "goodLevel": 0,
```

Comment:
you want all levels colored special here. but are all levels of hardened mana good? argumentative